build-tools/scripts/XABuildPaths.cs.in

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.IO;
+
+namespace Xamarin.Android.Build
+{
+	public static class Paths
+	{
+		public const string Configuration = "@CONFIGURATION@";
+		public const string TopDirectory = "@TOP_DIRECTORY@";
+
+		public static readonly string BinDirectory = Path.Combine (TopDirectory, "bin", Configuration);
```

Comment:
This should be `PrefixDirectory`; "bin" would imply `$prefix/bin` aka `{TopDirectory}/bin/{Configuration}/bin`.

Reply:
...though based on `XABuildScript`, this *should* be `BinDirectory`, and instead should be:

```csharp
public static readonly string BinDirectory = Path.Combine (TopDirectory, "bin", Configuration, "bin");
```

Reply:
Not sure.. $PREFIX is traditionally where you `make install` to...